src/test/spec/time-picker/time-picker.spec.ts

```diff
@@ -459,6 +459,12 @@ describe('TimePickerComponent', function(this: ITestContext) {
     expect(this.context.component.showNow).toBeTrue();
   });
 
+  it('should set showHourOptions via attribute', function(this: ITestContext) {
```

Comment:
This test is good, but you should probably also add one to check that they're actually excluded from the options, similar to the ['should show "now" as the first option'](https://github.com/tyler-technologies-oss/forge/blob/138f9291d78da4fc2717a277b1cdfff2120cdba5/src/test/spec/time-picker/time-picker.spec.ts#L952-L964) test but verifying that "now" is the only option if this property is false.

Reply:
thanks, added 2 additional tests.